src/core/thermostat.hpp

```diff
@@ -128,6 +128,11 @@ struct LangevinThermostat : public BaseThermostat {
       gamma_rotation = gamma;
     }
     pref_noise_rotation = sigma(kT, time_step, gamma_rotation);
+    // If gamma_magnet is not set explicitly, use the translational one.
+    if (gamma_magnet < GammaType{}) {
```

Comment:
If the LLG feature is not compiled in, the gamma_magnet doesn’t even exist, so I wonder if it’s better to use the preprocessor directives here also.